docs/configuration.md

```diff
@@ -0,0 +1,65 @@
+# Configuration
+
+Mutant can be configured with a config file that can be named one of the following: `.mutant.yml`, `config/mutant.yml`, or `mutant.yml`
+
+The following options can be configured through the config file:
+
+#### `includes`
+
+The includes key takes a list of strings to add to ruby's `$LOAD_PATH`. This is similar to ruby's `-I` flag.
+
+```yml
+---
+includes:
+  - lib
+```
+
+Additional includes can be added by providing the `-I` or `--include` flag to the CLI.
+
+#### `requires`
+
+The requires key takes a list of ruby files to be required. This is how mutant loads your application code.
+
+```yml
+---
+requires:
+  - my_app
+```
+
+Additional requires can be added by providing the `-r` or `--require` flag to the CLI.
+
+#### `integration`
+
+Specifies which mutant integration to use. If your tests are writen in [RSpec](https://rspec.info/), this should be set to `rspec`. If your tests are written in [minitest](https://github.com/seattlerb/minitest), this should be set to `minitest`.
+
+```yml
+---
+integration: rspec
+```
+
+The integration can be overridden by providing the `--use` flag to the CLI.
+
+#### `fail_fast`
+
+When `fail_fast` is enabled, mutant will stop as soon as it encounters an alive mutation. This can be helpful for incremental workflows where you want to see the output of a failed mutation immediately. Valid values are `true` and `false`.
+
+```yml
+---
+fail_fast: true
+```
+
+`fail_fast` cannot currently be overridden by the CLI.
+
+#### `jobs`
+
+Specifiy how many processes mutant uses to kill mutations. Defaults to the number of processors on your system.
+
+```yml
+---
+jobs: 8
+```
+
+The number of jobs can by overridden by the `-j` or `--jobs` flag in the CLI.
```

Comment:
`s/can by/can be/`

Reply:
Technically, if a CLI argument requires another argument its not a flag. `-j` is not self standing, its an option. Hence this should state:

```
> The number of jobs can be ovcerwritten by the `-j/--jobs` option in the CLI.

The word `flag` AFAIK describes `--version` and friends that consume no other argument.

Reply:
I have used / sometimes seen them used interchangeably but that's an interesting distinction. 👍